lerobot/templates/visualize_dataset_template.html

```diff
@@ -75,7 +75,7 @@ <h1 class="text-xl font-bold mt-4 font-mono">
             {% for video_info in videos_info %}
             <div class="max-w-96">
                 <p class="text-sm text-gray-300 bg-gray-800 px-2 rounded-t-xl truncate">{{ video_info.filename }}</p>
-                <video muted loop type="video/mp4" class="min-w-64" @canplay="videoCanPlay" @timeupdate="() => {
+                <video muted loop type="video/mp4" class="min-w-64" @canplaythrough="videoCanPlay" @timeupdate="() => {
```

Comment:
upon further inspection [canplaythrough](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/canplaythrough_event) is what we want